docs/current_docs/api/enumerations.mdx

```diff
@@ -9,6 +9,10 @@ import TabItem from '@theme/TabItem';
 
 # Enumerations
 
+:::important
+The information on this page is only applicable to Go, Python and TypeScript SDKs. Interfaces are not currently supported in the PHP SDK.
```

Comment:
```suggestion
The information on this page is only applicable to Go, Python and TypeScript SDKs. Enumerations are not currently supported in the PHP SDK.
```

Reply:
Thank you! I should not have missed that, thank you for catching it. :pray: